core/server/master/src/main/java/alluxio/master/throttle/SystemMonitor.java

```diff
@@ -417,7 +417,7 @@ private void checkAndBackPressure() {
       }
     }
 
-    mLastHeartBeatTimeMS = System.nanoTime();
+    mLastHeartBeatTimeMS = System.currentTimeMillis();
```

Comment:
see line: https://github.com/Alluxio/alluxio/pull/16341/files#diff-848821f337aaafde249302df886fc786ba4baf85df6224176ffcefb47a119635R333

It is used Milliseconds to compare

Reply:
if (mLastHeartBeatTimeMS != 0
        && (System.currentTimeMillis() - mLastHeartBeatTimeMS
        - mHeartBeatIntervalMS) >= mPitThresholdOverloaded.getTotalJVMPauseTimeMS()) {
      // This heartbeat time is far from the heartbeat time last time
      // Directly set the status to OVERLOADED
      // Since it is set to overloaded, skip the rest check.
      statusTransition = (mCurrentSystemStatus == SystemStatus.OVERLOADED)
          ? StatusTransition.UNCHANGED : StatusTransition.ESCALATE;
      mCurrentSystemStatus = SystemStatus.OVERLOADED;